arch/inst/Zk/aes32dsi.yaml

```diff
@@ -3,9 +3,12 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: aes32dsi
-long_name: No synopsis available.
+long_name:  AES final round decryption instruction for RV32.
 description: |
-  No description available.
+  Sources a single byte from rs2 according to bs. To this it applies the inverse AES
+  SBox operation, and XOR’s the result with rs1. Must always be implemented such
```

Comment:
Let's just keep all of the text from the spec here, from "This instruction sources..." to "...data being operated on."
As in `aes32dsmi`, below, put backquotes around `rs1`, `rs2`, `bs`.
Also, it's subtle, but use a single quote `'` instead of whatever is there now (`’`).